File: pyg_manage_web/src/main/webapp/js/Service/brandService.js
//自定义服务
app.service('brandService',function ($http) {
    //分页查询
    this.findPage=function (page,rows) {
        return $http.get('../brand/findPage.do?page='+ page + '&rows=' + rows);
    }
    //分页条件查询
    this.search=function (page,rows,entity) {
        return $http.post('../brand/search.do?page='+ page + '&rows=' + rows,entity);
    }
    //查询某个品牌
    this.findOne=function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }
    //批量删除
    this.dele=function (ids) {
        return $http.get('../brand/delete.do?ids='+ids);
    }
    //添加
    this.add=function (entity) {
        return $http.post('../brand/add.do',entity);
    }
    //修改
    this.update=function (entity) {
        return $http.post('../brand/update.do',entity);
    }

    //下拉列表数据
    this.selectOptionList=function(){
        return $http.get('../brand/selectOptionList.do');
    }
});

File: pyg_sellergoods_interface/src/main/java/com/pyg/sellergoods/service/BrandService.java
package com.pyg.sellergoods.service;

import com.pyg.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

/**
 *品牌接口
 */
public interface BrandService {
    //查询所有
    public List<TbBrand> findAll();
    //查询所有之分页查询
    public PageResult findPage(int pageNum , int pageSize);
    //添加
    public void add(TbBrand tbBrand);
    //修改的查询回现
    public TbBrand findOne(Long id);
    //修改
    public  void update(TbBrand tbBrand);
    //批量删除
    public void delete(Long[] ids);
    //品牌条件查询
    public PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);

    /**
     * 品牌下拉框数据
     */
    List<Map> selectOptionList();
}
